services/user/XAdmin/ui/src/configuration/utils.ts

```diff
@@ -276,7 +273,7 @@ export const mergeConfig = (
         ...updated,
         p2p: mergeSimple(prev.p2p, updated.p2p, user.p2p),
         producer: mergeSimple(prev.producer, updated.producer, user.producer),
-        host: mergeSimple(prev.host, updated.host, user.host),
+        hosts: mergeSimple(prev.hosts, updated.hosts, user.hosts),
```

Comment:
Should use mergeList

Reply:
I think a little more work will be needed there. `mergeArray` expects a list of objects.

Reply:
`mergeSimple` got the boot unstuck. But I believe I'm going to need to turn `hosts` into a list of objects anyway...for working with them in the configuration form. Still familiarizing myself with this part of the x-admin code base though.